class/defaults.yml

```
@@ -724,7 +724,7 @@ parameters:
               name: ${appuio_reporting:cluster_id_to_name_map:c-appuio-cloudscale-lpg-2}
           - product_id: appuiocloud-blockstorage-exoscalessd
             params:
-              storage_class: ssd.*
+              storage_class: ssd.*|rbd-storagepool-cluster
```

```suggestion
              storage_class: rbd-storagepool-cluster
```

There's no storage classes named `ssd.*` on Exoscale -- this looks like a copy-paste error.

---

💡